scripts/question_answering/config.py

```diff
@@ -0,0 +1,158 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=
+r"""
+This file contains all hyperparameters.
+"""
+import argparse
+
+import mxnet as mx
+
+parser = argparse.ArgumentParser(
+    description='The hyperparameters for QANet and optimizer.')
+
+# configure
+parser.add_argument('--use_gpu', type=bool, default=True,
+                    help='Whether use gpu for train and eval.')
+parser.add_argument('--gpu_list', type=str,
+                    default='1', help='List of gpu index.')
+parser.add_argument('--prefix_model', type=str, default='model_',
+                    help='The prefix name for model saving.')
+parser.add_argument('--prefix_trainer', type=str, default='trainer_',
+                    help='The prefix name for trainer saving.')
+parser.add_argument('--load_trained_model', type=bool,
+                    default=False, help='Whether load trained model.')
+parser.add_argument('--trained_model_name', type=str,
+                    default='', help='File name of trained model.')
+parser.add_argument('--trained_trainer_name', type=str,
+                    default='', help='File name of trained trainer.')
+parser.add_argument('--evaluate_interval', type=int, default=5000,
+                    help='Evaluate the model after how many batchs.')
+parser.add_argument('--is_train', type=bool, default=True,
+                    help='Whether train or evaluate the model.')
+parser.add_argument('--data_path', type=str,
+                    default='../data/', help='The root dir of data.')
+parser.add_argument('--train_file_name', type=str,
+                    default='train-v1.1.json', help='The train file name.')
+parser.add_argument('--dev_file_name', type=str,
+                    default='dev-v1.1.json', help='The dev file name.')
+parser.add_argument('--processed_train_file_name', type=str,
+                    default='train_sorted.json', help='The processed train file name.')
+parser.add_argument('--processed_dev_file_name', type=str,
+                    default='dev_sorted.json', help='The processed dev file name.')
+parser.add_argument('--glove_file_name', type=str,
+                    default='glove.840B.300d.txt', help='The glove file name.')

```

Comment:
This glove embedding is available in gluonnlp API with `gluonnlp.embedding.create('glove', 'glove.840B.300d')`, so that users don't have to download it manually.